### ISSUE TITLE
[Broken Website] Mercadolibre.com

### ISSUE THREAD

**Issue by alfreav** [darkreader/darkreader#2983]
**MercadoLibre**
https://www.mercadolibre.com.ar/

**System info:**
 - OS: [Windows 10]
 - Browser: [Edge Chromium 83.0.478.56]
 - Darkreader Version: [4.9.13]
 
Fullscreen 
![screenshot-www mercadolibre com ar-2020 06 30-17-47-15](https://user-images.githubusercontent.com/28546425/86175679-27c47c00-bafa-11ea-88c3-6abf90752176.png)

Windowed search result
![screenshot-listado mercadolibre com ar-2020 06 30-17-48-10](https://user-images.githubusercontent.com/28546425/86175714-36129800-bafa-11ea-8cc6-9d8f4719169b.png)

TY.

**Comment by Gusted** [comment]
Yeah fill out Actuall result and Expected result I have no idea where I need to look for. And steps to reproduce will be nice too..

**Comment by alfreav** [comment]
> Yeah fill out Actuall result and Expected result I have no idea where I need to look for. And steps to reproduce will be nice too..

Sorry. I added some info.

**Comment by Gusted** [comment]
We need to add support to respect `@media` values and add them in their selector.
